### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by watson**
I thought this module needed a few extra tests due to the complicated and messy code-base. Or maybe I just felt bored and thought it would be an interesting experiment to figure out how to test something like this :wink:

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -4,5 +4,7 @@ var MAX_SAFE_INTEGER = require('./');
 
 test(function (t) {
 	t.assert(typeof MAX_SAFE_INTEGER === 'number');
+	t.is(MAX_SAFE_INTEGER + 1, MAX_SAFE_INTEGER + 2);
```
Comment (shinnn) [inline test.js:7]: `ava` needs to be updated to `"*"` if you want to use `t.is`. https://github.com/sindresorhus/max-safe-integer/blob/e481404081842afb02efd847a03c4efa190551ea/package.json#L36

Comment (watson) [inline test.js:7]: haha... this is what happens when I make one-shot PR's by editing the file directly on Github ;) I'll update the PR

**Comment by sindresorhus** [comment]
Good stuff. Fixed it for you. Thanks :)

**Comment by watson** [comment]
@sindresorhus thanks :)

**Pull request closed without merging**